code/game/objects/items/reagent_containers/food/snacks/grown.dm

```diff
@@ -75,6 +75,61 @@
 	plantname = "poppies"
 	black_market_value = 25
 
+/obj/item/reagent_container/food/snacks/grown/nettle
+	plantname = "nettle"
+	desc = "It's probably <B>not</B> wise to touch it with bare hands..."
+	icon = 'icons/obj/items/weapons/weapons.dmi'
+	name = "nettle"
+	icon_state = "nettle"
+	damtype = "fire"
+	force = 15
+	flags_atom = NO_FLAGS
+	throwforce = 1
+	w_class = SIZE_SMALL
+	throw_speed = SPEED_FAST
+	throw_range = 3
+
+	attack_verb = list("stung")
+	hitsound = ""
+
+	var/potency_divisior = 5
+
+/obj/item/reagent_container/food/snacks/grown/nettle/Initialize()
+	. = ..()
+	force = round((5+potency/potency_divisior), 1)
+
+/obj/item/reagent_container/food/snacks/grown/nettle/pickup(mob/living/carbon/human/user, silent)
+	. = ..()
+	if(istype(user) && !user.gloves)
+		to_chat(user, SPAN_DANGER("The nettle burns your bare hand!"))
+		if(istype(user, /mob/living/carbon/human))
+			var/organ = ((user.hand ? "l_":"r_") + "arm")
+			var/obj/limb/affecting = user.get_limb(organ)
+			if(affecting.take_damage(0, force))
+				user.UpdateDamageIcon()
+		else
+			user.take_limb_damage(0, force)
+		return TRUE
+	return FALSE
```

Comment:
```suggestion
/obj/item/reagent_container/food/snacks/grown/nettle/pickup(mob/living/carbon/human/user, silent)
	. = ..()
	if(!istype(user) || user.gloves)
		return FALSE

	to_chat(user, SPAN_DANGER("The nettle burns your bare hand!"))
	var/obj/limb/affecting = user.get_limb(user.hand ? "l_hand":"r_hand")
	affecting.take_damage(0, force)
	return TRUE
```

irrc lines 110 & 111 will never execute. istype(user) is just istype(user, mob/living/carbon/human), and for that condition to pass the one on line 105 will always pass too.